Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -444,6 +444,8 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
             {
                 var audioCodec = state.ActualOutputAudioCodec;
                 var videoCodec = state.ActualOutputVideoCodec;
+                var hardwareAccelerationType = _serverConfigurationManager.GetEncodingOptions().HardwareAccelerationType;
+                hardwareAccelerationType = string.IsNullOrEmpty(hardwareAccelerationType) ? "none" : hardwareAccelerationType;
```

Comment:
I'm not a server dev, but usually the API simply returns `null` when something isn't set, which is likely better than returning a string here.

Do wait for someone from the server team to confirm this, though.

Reply:
Additionally using an enum is preferred so the client devs know the possible values.

Reply:
I concur, enum if possible, null if not set.
I would give more input but currently on vacation without laptop

Reply:
Uses an enum now, looks like this:
![image](https://user-images.githubusercontent.com/10103146/124524780-300c6280-ddb1-11eb-8bff-265a94ff5002.png)